### DISCUSSION TITLE
Stretch Align Block

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#549] [Ideas]
Could you please add a **Stretch Align** block to the **Stretch** extension? This would expand the capabilities of the extension and allow users to create more complex projects! (Example: Loading bar for a game etc...)

![scratchblocks-2](https://github.com/TurboWarp/extensions/assets/135789443/b50b2f13-8147-48bc-9cd1-e2cfe7b45f3f)

**Comment by CST1229** [comment]
Stretch stretches with the costume/rotation center as the middle (just like vanilla Scratch's size blocks), so you could just move your costumes.

**Reply by BlueDome77** [reply]
Ok!
